trigger/advanced-automated-retries.yml

```diff
@@ -0,0 +1,13 @@
+name: Advanced Automated Retries
+source: advanced-automated-retries.rego
+type: trigger
+description: |
+  Sometimes Terraform or Pulumi deployments fail for a reason that has nothing to do with the code
```

Comment:
Let's remove the mention of Terraform or Pulumi here because this policy is general enough that it applies to all stacks.